examples/react/with-provider/src/DevCycleExample.tsx

```diff
@@ -7,13 +7,18 @@ export default function DevCycleExample(): React.ReactElement {
     const variableKeyString = 'variable-key-string'
     const variableKeyNumber = 'variable-key-number'
     const variableKeyBoolean = 'variable-key-boolean'
-    const variableKeyJson = 'json-key-string'
+    const variableKeyJsonString = 'variable-json-key-string'
 
     const variable = useVariable( variableKey, true)
-    const variableString = useVariable( variableKeyString, 'pink')
-    const variableNumber = useVariable( variableKeyNumber, 100)
+    const variableString = useVariable( variableKeyString, 'green')
+    const variableNumber = useVariable( variableKeyNumber, 600)
     const variableBoolean = useVariable( variableKeyBoolean, false)
-    const variableJson = useVariable( variableKeyJson, { 'jsonStringKey1':'json string value 1' })
+    //const variableJson = useVariable( variableKeyJson, { 'jsonStringKey1':'json string value 1' })
+     const variableJsonString = useVariable( variableKeyJsonString, { 'jsonStringKeyDefault':'json string value default' })
```

Comment:
why change the name to `variableJsonString` if it's not a string?

Reply:
because the Json variable value in this case is string; can be number, boolean as well; make sense for automation testing